src/turnkeyml/common/build.py

```diff
@@ -297,10 +328,52 @@ def get_wmic_info(command):
                 info_dict["Physical Memory"] = f"{mem_info_gb} GB"
             except ValueError:
                 info_dict["Physical Memory"] = mem_info_bytes
+            info_dict["BIOS Version"] = get_wmic_info("wmic bios get smbiosbiosversion")
+            info_dict["CPU Max Clock"] = (
+                get_wmic_info("wmic cpu get MaxClockSpeed") + "MHz"
+            )
         else:
             info_dict["Processor"] = "Install WMIC to get system info"
             info_dict["OEM System"] = "Install WMIC to get system info"
             info_dict["Physical Memory"] = "Install WMIC to get system info"
+            info_dict["BIOS Version"] = "Install WMIC to get system info"
+            info_dict["CPU Max Clock"] = "Install WMIC to get system info"
+
+        # Get driver versions
+        device_names = [
+            "NPU Compute Accelerator Device",
+            "AMD-OpenCL User Mode Driver",
+        ]
+        driver_versions = {}
+        for device_name in device_names:
+            driver_version = get_windows_driver_version(device_name)
+            driver_versions[device_name] = (
+                driver_version if driver_version is not None else "DRIVER NOT FOUND"
+            )
+        info_dict["Driver Versions"] = driver_versions
+
+        # Get NPU power mode
+        try:
```

Comment:
We might want to do this check only for AMD platforms

Reply:
OK.  I will check if "AMD" is in the Processor Name string.